client/src/components/App/NotificationSnackbar.js

```diff
@@ -1,11 +1,7 @@
 import React, { PureComponent } from 'react';
-import CheckCircleIcon from '@material-ui/icons/CheckCircle';
-import ErrorIcon from '@material-ui/icons/Error';
-import InfoIcon from '@material-ui/icons/Info';
 import CloseIcon from '@material-ui/icons/Close';
 import { amber, green } from '@material-ui/core/colors';
 import IconButton from '@material-ui/core/IconButton';
-import WarningIcon from '@material-ui/icons/Warning';
```

Comment:
Thank you for removing this! The warnings in the terminal were getting annoying lol